src/Scrumptious.Service/Startup.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Scrumptious.Service
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. ([csharpsquid:S1186](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&issues=AWVI3qEHiZQY_aXE0fBU&open=AWVI3qEHiZQY_aXE0fBU)

[ISSUE_KEY]: AWVI3qEHiZQY_aXE0fBU